docs/starters.yml

```diff
@@ -1,20 +1,3 @@
-- url: https://gatsby-starter-lamma.netlify.app/
```

Comment:
Why are all these starters being deleted? Is this intentional?

Reply:
this is unintentional, should i close this ?